.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
For eslint rules, plz use: https://github.com/casdoor/casdoor/blob/master/web/.eslintrc